src/config/Config.zig

```diff
@@ -560,6 +560,7 @@ palette: Palette = .{},
 /// On macOS, background opacity is disabled when the terminal enters native
 /// fullscreen. This is because the background becomes gray and it can cause
 /// widgets to show through which isn't generally desirable.
+/// On macOs, this setting cannot be reloaded and needs a restart
```

Comment:
```suggestion
/// On macOS, changing this configuration requires restarting Ghostty completely.
```
would echo the spelling of `macOS` used just above, and the verbiage used for e.g. `quick-terminal-position` :)

Reply:
Thanks @akx ! I wrote macOs, but not sure if this is specific, may be necessary for linux as well, I'll try to get it tested, but if anyone knows how this goes on linux, I'd appreciate the inputs!